lib/experience-bootstrap.js

```diff
@@ -0,0 +1,30 @@
+const {
+  loadConfig,
+  logResult,
+  logError,
+  hasBootstrapped
+} = require('./utils');
+const getApi = require('./get-api');
+const { experience } = require('./get-download-params');
+const getDownloader = require('./get-downloader');
+const downloader = getDownloader(experience);
+
+module.exports = async (cmd, loadedConfig, applicationInfo) => {
+  const { apiToken, applicationId, applicationName } = loadedConfig ? loadedConfig : await loadConfig();
+  const api = await getApi({ apiToken });
+  try {
+    applicationInfo = applicationInfo || await api.application.get({ applicationId });
+    const bootstrap = hasBootstrapped(applicationInfo);
+    if (bootstrap) {
+      return logResult('Cannot Complete', `Bootstrapping has already been ${bootstrap.status} for ${applicationName}`, 'yellow');
+    }
+    const results = await api.experience.bootstrap({ applicationId });
+    await downloader(null, {}, { apiToken, applicationId });
+    logResult('Experience URL', `${applicationInfo.endpointSlug}`);
```

Comment:
this is not the URL, this is just the slug - you need to build the full URL they can copy paste